harness/determined/cli/master.py

```diff
@@ -89,19 +97,60 @@ def logs(args: Namespace) -> None:
         print(format_log_entry(response.logEntry))
 
 
+def parseLogLevel(level: str) -> bindings.v1LogLevel:
+    if level.upper() == "INFO":
+        return bindings.v1LogLevel.INFO
+    elif level.upper() == "TRACE":
+        return bindings.v1LogLevel.TRACE
+    elif level.upper() == "DEBUG":
+        return bindings.v1LogLevel.DEBUG
+    elif level.upper() == "WARNING" or level.upper() == "WARN":
+        return bindings.v1LogLevel.WARNING
+    elif level.upper() == "ERROR":
+        return bindings.v1LogLevel.ERROR
+    elif level.upper() == "CRITICAL":
+        return bindings.v1LogLevel.CRITICAL
+    else:
+        raise CliError(
+            "Error: Invalid Log level provided. "
+            + "Acceptable levels are: INFO, TRACE, DEBUG, WARNING, ERROR and CRITICAL."
+        )
+
+
 # fmt: off
 
 args_description = [
     Cmd("master", None, "manage master", [
-        Cmd("config", config, "fetch or patch master config", [
-            Arg("--log", action="store_true",
-                help="patch log in master config"),
-            Arg("--level", type=str, default=None, required=False,
-                help="set log level in the master config"),
-            Arg("--color", type=str, default=None, required=False,
-                help="set log color in the master config"),
-            Group(cli.output_format_args["json"], cli.output_format_args["yaml"])
-        ]),
+        Cmd(
+            "config",
+            None,
+            "manage master config",
+            [
+                Cmd(
+                    "show",
+                    config,
+                    "show master config",
+                    [
+                        Group(cli.output_format_args["json"],
+                              cli.output_format_args["yaml"])
+                    ],
+                    is_default=True,
+                ),
+                Cmd(
+                    "set",
+                    set_master_config,
+                    "set master config",
+                    [
+                        Arg("--log.level", type=str, default=None, required=False,
```

Comment:
you can set `dest` here, which will assign the argument to whatever name you choose and pass it down to `argparse`. 

`dest=log_level` would allow you to access `args.log_level`

Reply:
done